streaming/index.js

```diff
@@ -1351,15 +1351,23 @@ const startServer = async () => {
  * @param {function(string): void} [onSuccess]
  */
 const attachServerWithConfig = (server, onSuccess) => {
-  if (process.env.SOCKET || process.env.PORT && isNaN(+process.env.PORT)) {
-    server.listen(process.env.SOCKET || process.env.PORT, () => {
+  if (process.env.SOCKET) {
+    server.listen(process.env.SOCKET, () => {
```

Comment:
For some reason we supported either SOCKET or PORT as a non-port here for unix sockets, whilst puma did not. This aligns the two.